packages/@okta/vuepress-site/docs/reference/api/authn/index.md

```diff
@@ -624,7 +621,6 @@ Authenticates a user through a [trusted application](#trusted-application) or pr
 
 **Notes:**
```

Comment:
This is not the typical way to format **Notes**.
We usually put the note around a note-text-box. Do you want to do this?
Otherwise, you should just use a regular H5 header format for this heading. For example:
```
##### Notes
```